File: iOS/EvianBlog/api/resume/ResumeAPI.swift
//
//  ResumeAPI.swift
//  EvianBlog
//
//  Created by Evian张 on 2020/5/18.
//  Copyright © 2020 Evian张. All rights reserved.
//

import Alamofire

import Foundation
import Combine

extension BlogAPI {
	func getResume() -> AnyPublisher<String, BlogAPIError> {
		self.fetch(url: Endpoint.Resume.getResume)
	}
}

File: Android/app/src/main/java/top/evian_zhang/evianblog/api/resume/ResumeAPI.kt
package top.evian_zhang.evianblog.api.resume

import top.evian_zhang.evianblog.api.BlogAPI
import top.evian_zhang.evianblog.api.Endpoint

suspend fun BlogAPI.getResume(): String {
    return this.fetch(Endpoint.Resume.getResume)
}
